drivers/ethernet/eth_utils.h

```diff
@@ -0,0 +1,11 @@
+/* Ethernet driver utility functions
+ *
+ * Copyright (c) 2017 Linaro Ltd
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <kernel.h>
+
+void eth_ipv6_mcast_to_mac_addr(const struct in6_addr *ipv6_addr,
```

Comment:
don't put this in drivers, that should go to include/net/ethernet.h and the implementation in subsys/net/ip/l2/ethernet.c

Reply:
Done.